### PR TITLE
WinAPI cleanup

### PR DESCRIPTION
**Pull request by airelil**
Explicit signatures for WinAPI calls.
Move local declarations of external Win API funcs from HwndWrapper to win32functions
Always call GetWindowThreadProcessId with proper HWND type instead of int

TBD: move all win API declarations from win32hook to win32functions

### CODE REVIEW

**Comment by vasily-v-ryabov** [comment]
This pull request **introduces 2 alerts** when merging fad0126b8c3b4366140884feea1fa4339e40bda7 into d2e1b1a96aa198e86abedd9efe798d5b1dc5e5e8 - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-3f7b151f66dd9f59b1c04ead71a982614c07bcb3)

**new alerts:**

* 2 for Unused import

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 3 alerts** when merging a3ff171d0e9c5d1a4eddf3190b4e118163f3b99b into d2e1b1a96aa198e86abedd9efe798d5b1dc5e5e8 - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-2617bf7950064e667b9c44734041c5c82e923138)

**fixed alerts:**

* 2 for Variable defined multiple times
* 1 for Module is imported with &#39;import&#39; and &#39;import from&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: pywinauto/win32functions.py [hunk 1]
```diff
@@ -31,250 +31,672 @@
...
+from ctypes import byref
+from ctypes import POINTER
+from ctypes import c_ubyte
+from ctypes import c_size_t
+from ctypes import cdll
 
 import sys
 if sys.platform == "cygwin":
-    windll = ctypes.cdll
+    cygdll = cdll
```
Comment (vasily-v-ryabov) [inline pywinauto/win32functions.py]: `cygdll` is not used. Do we really need this rename?

File: pywinauto/win32functions.py [hunk 2]
```diff
@@ -355,7 +777,7 @@ def LoWord(value):
 def WaitGuiThreadIdle(handle):
     """Wait until the thread of the specified handle is ready"""
     process_id = wintypes.DWORD(0)
-    GetWindowThreadProcessId(handle, ctypes.POINTER(wintypes.DWORD)(process_id))
+    GetWindowThreadProcessId(handle, POINTER(wintypes.DWORD)(process_id))
```
Comment (vasily-v-ryabov) [inline pywinauto/win32functions.py]: This call is different than other calls of this function. Maybe it makes sense to align them (using `byref`, for example).

File: pywinauto/win32functions.py (hunk 1 above)
Comment (airelil) [reply pywinauto/win32functions.py]: This reference to cygwin was added long time ago by Mark, but I also don't see how it was used. All the API declarations below always refered to ```ctypes.windll``` explicitly. I'll drop this code for now. Cygwin support, if it's still required, will need a separate issue.

File: pywinauto/win32functions.py (hunk 2 above)
Comment (airelil) [reply pywinauto/win32functions.py]: Should be fixed now with a299ab6

**Comment by vasily-v-ryabov** [comment]
This pull request **fixes 3 alerts** when merging a299ab61d34bf97c8ca34ea348e11f2fe6e0b9de into b00af57da4e0f748f2bd71b514c27928363245aa - [view on LGTM.com](https://lgtm.com/projects/g/pywinauto/pywinauto/rev/pr-7cd9ddcdec5326fbe9df2134466a9f328d51559f)

**fixed alerts:**

* 2 for Variable defined multiple times
* 1 for Module is imported with &#39;import&#39; and &#39;import from&#39;

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**